[OV JS] Add link to live sample of openvino-node usage (#24973)
openvinotoolkit/openvino: src/bindings/js/node/README.md

- [OpenVINO™ Node.js API](https://docs.openvino.ai/2024/api/nodejs_api/nodejs_api.html)
- [OpenVINO™ Node.js Bindings Examples of Usage](https://github.com/openvinotoolkit/openvino/blob/master/samples/js/node/README.md)

> ## Live sample
> 
> This sample you can run in the browser, no installation required.
> [Codesandbox](https://codesandbox.io/) is free online service with limited resources. For the best performance and more control, you should run the sample locally.

Reviewer: ```suggestion
## Live Sample

You can run this sample in the browser; no installation is required.
[Codesandbox](https://codesandbox.io/) is a free online service with limited resources. For optimal performance and more control,  it is recommended to run the sample locally.
```
